pkg/distributor/forwarding/forwarding.go

```diff
@@ -432,6 +434,7 @@ func (f *forwarder) submitForwardingRequest(ctx context.Context, endpoint string
 	req.requestWg = requestWg
 
 	// Target endpoint and TimeSeries to forward.
+	req.user = user
```

Comment:
the comment above this line is now strange because it only talks about endpoint/timeseries

Reply:
I've changed the comment to "Request parameters". WDYT?